samples/Sample.RabbitMQ.SqlServer/Sample.RabbitMQ.SqlServer.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Dapper" Version="2.0.35" />
```

Comment:
Can this reference be removed? It doesn't seem to be used

Reply:
![image](https://user-images.githubusercontent.com/12774053/84354245-1ec74580-abf3-11ea-9e3c-2522954b5c05.png)

Reply:
Thank you for pointing, can we modify the example to use ado.net?